go/cmd/betabot/main.go

```diff
@@ -93,11 +104,19 @@ func betabot() error {
 			}
 
 			// parse
-			err = json.Unmarshal(data, &bot.store)
+			pstore := StoreJSON{}
+			err = json.Unmarshal(data, &pstore)
 			if err != nil {
 				return fmt.Errorf("parse %q: %w", *storePath, err)
 			}
 			bot.isReplaying = true // if the db exists on disk, then we switch the bot to replay mode
+			bot.store.StaffConvPK = pstore.StaffConvPK
+			i := len(pstore.Convs)
+			bot.store.Convs = make([]*Conversation, i)
+			for i != 0 {
+				i--
+				bot.store.Convs[i] = &Conversation{ConversationJSON: pstore.Convs[i]}
+			}
```

Comment:
I easily imagine there is a good reason to do this but I don't have enough knowledge in go to see what, can you add a quick comment explaining why you make an intermediate variable and manual assignment (step by step deepcopy) instead of just unmarshaling to the target?

Reply:
> I easily imagine there is a good reason to do this but I don't have enough knowledge in go to see what, can you add a quick comment explaining why you make an intermediate variable and manual assignment (step by step deepcopy) instead of just unmarshaling to the target?

@moul I don't know it myself :D (Go vet hates {M,Unm}arshal of values with locker in them). (I'm still gonna add the comment.)